engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-products.controller.js

```diff
@@ -6,41 +6,92 @@
  * @requires translate
  * @requires ContentHost
  * @requires Product
- * @requires CurrentOrganization
  *
  * @description
  *   Provides the functionality for the content-host products action pane.
  */
 angular.module('Bastion.content-hosts').controller('ContentHostProductsController',
-    ['$scope', 'translate', 'ContentHost', 'Product', 'CurrentOrganization',
-    function ($scope, translate, ContentHost, Product, CurrentOrganization) {
+    ['$scope', 'translate', 'HostSubscription',
+    function ($scope, translate, HostSubscription) {
+        var defaultOverride = "default";
 
         $scope.successMessages = [];
         $scope.errorMessages = [];
         $scope.displayArea = { working: true, isAvailableContent: false };
 
-        $scope.isAnyAvailableContent = function (products) {
-            var isAvailableContent = false;
-            angular.forEach(products, function (product) {
-                if (product['available_content'].length > 0) {
-                    isAvailableContent = true;
-                }
-            });
-            return isAvailableContent;
+        $scope.getEnabledText = function (enabled, overrideEnabled) {
+            var enabledText;
+            overrideEnabled = overrideEnabled + "";
+
+            if (overrideEnabled === defaultOverride) {
+                enabledText = enabled ? translate("Yes (Default)") : translate("No (Default)");
+            } else if (overrideEnabled === "1") {
+                enabledText = translate("Override to Yes");
+            } else {
+                enabledText = translate("Override to No");
+            }
+
+            return enabledText;
         };
 
-        $scope.contentHost.$promise.then(function () {
-            ContentHost.products({id: $scope.contentHost.uuid,
-                                  'organization_id': CurrentOrganization,
-                                  enabled: true,
-                                  'full_result': true,
-                                  'include_available_content': true
-                                 }, function (response) {
-                $scope.products = response.results;
-                $scope.displayArea.isAvailableContent = $scope.isAnyAvailableContent($scope.products);
-                $scope.displayArea.working = false;
+        HostSubscription.productContent({id: $scope.$stateParams.hostId,
+                              'full_result': true,
+                              'include_available_content': true
+                             }, function (response) {
```

Comment:
The formatting here is very difficult to follow.  What about declaring the params outside of the call to `productContent()`?